src/nifgen/examples/nifgen_trigger_example.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+import nifgen
+import sys
+import time
+
```

Comment:
Other examples in [nifgen](https://github.com/ni/nimi-python/tree/master/src/nifgen/examples) don't have `_example` suffix in their file name. Let's be consistent.